libbeat/scripts/Makefile

```diff
@@ -51,7 +51,7 @@ build: $(GOFILES)
 # Create test coverage binary
 .PHONY: buildbeat.test
 buildbeat.test: $(GOFILES)
-	go test -c -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
+	go test -c -race -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
```

Comment:
This slows down `make test` a bit. For example, in Packetbeat, `make test` takes 119 seconds compared to 74 in master (best out of two).  It's not a huge increase but I'm thinking we should have a follow up to make system tests be able to run with the non-testing binary.